src/common/message-creators/card-selection-message-creator.ts

```diff
@@ -8,6 +8,7 @@ export interface CardSelectionMessageCreator {
         ruleId: string,
         resultInstanceUid: string,
         event: React.SyntheticEvent,
+        testKey?: string,
     ) => void;
     toggleRuleExpandCollapse: (ruleId: string, event: React.SyntheticEvent) => void;
```

Comment:
These other interface methods will have to be updated with an optional testKey parameter too, see my comment in the assessment card selection message creator for context.